crm/ebay/manage.rst

```diff
@@ -0,0 +1,31 @@
+======================
+How to list a product?
+======================
+
+Listing without variation
+=========================
+
+In order to list a product, you need to check the **use eBay** field on a product form. Then the eBay tab will be available.
+
+.. image:: ./media/ebay_manage_1.png
+  :align: center
+
+If you check the **Use Stock Quantity** field, the quantity sets on eBay will be the Odoo **Forecast Quantity**.
```

Comment:
we try to avoid using "you" when possible. So in this case, you rephrase "When the **Use Stock Quantity** field is  checked..."